python/python/bystro/api/cli.py

```diff
@@ -280,7 +280,7 @@ def login(args: argparse.Namespace, print_result=True) -> CachedAuth:
     return state
 
 
-def authenticate(args) -> tuple[CachedAuth, dict]:
+def authenticate(dir_path: str) -> tuple[CachedAuth, dict]:
```

Comment:
Let's name dir_path something a bit more idiomatic (bystro_token_path or something like that), and set a default (~/.bystro).

THen, let's make sure the bystro_token_path exists or make it. the simplest way is to os.path.mkdir(bystro_token_path , exists_ok=True)

Reply:
Please make the default for this DEFAULT_DIR